webpack/js/modules.js

```diff
@@ -0,0 +1,47 @@
+module.exports = (ENV, { SRC }) => {
+  const eslintLoader = (ENV === 'production')
+    ? {
+      test: /\.jsx?$/,
+      enforce: 'pre',
+      loader: 'eslint-loader',
+      options: {
+        filePath: SRC,
+      },
+    }
+    : null;
+
+  return {
+    rules: [
+      eslintLoader,
+      {
+        // .js and .jsx files are caught
+        test: /\.jsx?$/,
+        exclude: /(node_modules|thirdparty)/,
+        loader: 'babel-loader',
+        options: {
+          presets: [
+            ['es2015', { modules: false }],
```

Comment:
Should this be `es2015-webpack` now? https://gist.github.com/sokra/27b24881210b56bbaff7#babel-and-webpack